logging/logging.go

```diff
@@ -923,3 +928,41 @@ func printEntryToWriter(entry *logpb.LogEntry, w io.Writer) error {
 	}
 	return err
 }
+
+func (l *Logger) redirectAsJSON(pbe *logpb.LogEntry) (err error) {
+	err = serializeEntryToWriter(pbe, l.redirectOutputWriter)
+	if err == nil {
```

Comment:
Flip this logic, keep happy path left justified.

Reply:
Applies below too.

Reply:
The goal was to have a single return. Will flip :+1: